src/components/StateEligibility.js

```diff
@@ -48,6 +48,26 @@ export default function StateEligibility() {
                 </AccordionSummary>
                 <AccordionDetails className={classes.accordionDetails}>
                     <List>
+                        <EligibilityGroupItem>
+                            <a
+                                href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases"
```

Comment:
Eventually it'd be nice to point this to the equivalent education-specific link (like how below we link to info for each specific group) but until we know that url, this is a good placeholder.

Reply:
Yes, that's the intent!